pychromecast/controllers/multizone.py

```diff
@@ -31,22 +161,44 @@ def _add_member(self, uuid, name):
             self._members[uuid] = name
             _LOGGER.debug("(%s) Added member %s(%s), members: %s",
                           self._uuid, uuid, name, self._members)
-            if self._callback:
-                self._callback.multizone_member_added(uuid)
+            for listener in self._status_listeners:
+                listener.multizone_member_added(uuid)
 
     def _remove_member(self, uuid):
         name = self._members.pop(uuid, '<Unknown>')
         _LOGGER.debug("(%s) Removed member %s(%s), members: %s",
                       self._uuid, uuid, name, self._members)
-        if self._callback:
-            self._callback.multizone_member_removed(uuid)
+        for listener in self._status_listeners:
```

Comment:
let's make a copy to prevent mutated while iteration:

```suggestion
        for listener in list(self._status_listeners):
```

Reply:
Fixed, along with a couple more same.